drivers/dht/dht.c

```diff
@@ -126,8 +125,8 @@ int dht_init(dht_t *dev, const dht_params_t *params)
 
 int dht_read(dht_t *dev, int16_t *temp, int16_t *hum)
 {
-    uint16_t csum;
-    uint16_t raw_hum, raw_temp;
+    uint8_t csum;
+    uint8_t raw_temp_I, raw_temp_D, raw_hum_I, raw_hum_D;
```

Comment:
Note that in C uppercase is usually exclusively used for preprocessor macros, which use all upper case

```suggestion
    uint8_t raw_temp_i, raw_temp_d, raw_hum_i, raw_hum_d;
```